### DISCUSSION TITLE
StreamFX using OBS via Portable Apps

### DISCUSSION THREAD

**Discussion by iNgeon** [Xaymar/obs-stream-effects#990] [General]
I have a question on installing StreamFX with OBS Portable. I prefer using PortableApps because it can update and less stuff in registry. 
I use OBS studio with the platform https://portableapps.com/apps/music_video/obs-studio-portable
If i run the Stream FX installer, I select portable and then its asks for install location (like today used StreamFX 0.12.0 Beta 164)
I replaced C:\Users\Username\AppData\Local\Programs\obs-studio\plugins\StreamFX with C:\Port\PortableApps\OBSPortable\App\obs-studio\StreamFX
I then have to manually move the \data & \obs-plugins into \App\obs-studio\ which I assume is correct.
However, launching OBS it seems StreamFX doesn't hook.
Any advice would be appreciated. I tried searching, maybe i didnt search well enough. 

[Folder structure OBS portable.txt](https://github.com/Xaymar/obs-StreamFX/files/10146345/Folder.structure.OBS.portable.txt)

**Comment by ReaperzxxTJ** [comment]
im working with obs 28 still (stream FX 164 is for OBS 29 beta)
i can easily install Stream FX 134 but 151 doesn't install into portable mode correctly

if this is how it will be going forward id love to know how to install to portable mode

**Comment by Richie-M** [comment]
Having the same issue, install to the location of my portable OBS as usual. It creates the Uninstall in that location (which works) but whne i start OBS, StreamFX does not load.

**Reply by Richie-M** [reply]
just checked /data/obs-plugins/StreamFX/ has been created properly.
and /obs-plugins/64bit/ contain StreamFX.dll & .pdb

Used streamfx-windows-0.12.0a134-g6853cc6a.exe and it installs fine. So definitely an issue with latest update

**Reply by Xaymar** [reply]
System Requirements are on each version of StreamFX. They're not written there as a joke, they're requirements.

**Reply by Richie-M** [reply]
> System Requirements are on each version of StreamFX. They're not written there as a joke, they're requirements.

thanks for pointing out 

> Since [OBS Studio v0.15.0](https://github.com/obsproject/obs-studio/releases/tag/0.15.0) it is possible to load plugins from %ProgramData%\obs-studio\plugins, which is a much safer option and now the only supported System-wide installation mode for StreamFX.

**Reply by Xaymar** [reply]
> OBS Version | Minimum | 29.0.0-beta2

**Comment by Xaymar** [comment]
For further help, please join [the official Discord](https://s.xaymar.com/streamfx-dc) and create a thread under #support. Thank you.
